core/src/test/kotlin/org/evomaster/core/problem/rest/OpenAPILocalURLIssueTest.kt

```diff
@@ -0,0 +1,142 @@
+package org.evomaster.core.problem.rest
+
+import org.evomaster.core.remote.SutProblemException
+
+import io.swagger.v3.oas.models.OpenAPI
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+/*
+Testing the local URL issue with OpenAPI, 4 test cases:
+- a local file which exists and a valid URL
+- a local file which does not exist and a valid URL
+- a local file which exists and an invalid URL
+- a local file which does not exist and an invalid URL
+- a local file which is not a valid swagger but a valid JSON
+- a local file which is not a valid JSON
+- an invalid url
```

Comment:
the original issue in #701 was `IllegalArgumentException: URI path component is empty`. does any of these tests reproduce it?

Reply:
Hi Andrea,

Thanks for your feedback. I executed Evomaster with the following parameters:

--blackBox true --bbSwaggerUrl file://openapi_pet.json --bbTargetUrl "" http://remote-host/api/v1/microservice --header0 "Authorization: Bearer " --outputFormat JAVA_JUNIT_5 --outputFolder sample --maxTime 30s --ratePerMinute 60

I made sure that file://openapi_pet.json is also in the same directory as the executable. It did not provide the error message: "URI path component is empty", but it provided the error message "URI has an authority component". I am investigating the difference.